pallets/stableswap/src/types.rs

```diff
@@ -36,6 +45,7 @@ pub struct PoolInfo<AssetId, BlockNumber> {
 	pub initial_block: BlockNumber,
 	pub final_block: BlockNumber,
 	pub fee: Permill,
+	pub pegs_info: Option<PegUpateInfo<BlockNumber>>,
```

Comment:
we dont want to store this here. 

in fact , we dont need to preserve the fee across the blocks

i would put the updated_at info inside PoolPegs storage, basically alongside the peg current values. that's info is more relevant there. 

and the fee can be temporary storage for just the one block and cleared afterwards.